pages/api/studentPortal/profile/profilePicturesClient.ts

```diff
@@ -25,8 +27,18 @@ export const fetchProfilePicture = async (
     .send(new ListObjectsV2Command(params))
     .then(async (res) => {
       if (res.Contents && res.Contents.length > 0) {
-        const firstImage = res.Contents.filter((x) => !x.Key?.endsWith("/"))[0]
-          .Key;
+        const bucketObjects = res.Contents.filter((x) => !x.Key?.endsWith("/"));
+
+        // sort by last modified date
+        bucketObjects.sort((a, b) => {
```

Comment:
if student has uploaded multiple images to s3, select the most recently modified file to be their profile picture